src/main/java/com/iexec/worker/dataset/DataService.java

```diff
@@ -115,21 +113,21 @@ public void downloadStandardInputFiles(String chainTaskId, @Nonnull List<String>
     /**
      * Download a file from a URI in the provided parent
      * directory and save it with the provided filename.
-     * 
-     * @param chainTaskId Task ID, for logging purpose
-     * @param uri URI of  single file to download
+     *
+     * @param chainTaskId         Task ID, for logging purpose
+     * @param uri                 URI of  single file to download
```

Comment:
Is the extra space you added here a convention, or is there another reason for it?

Reply:
> Is the extra space you added here a convention, or is there another reason for it?

It's surely the IDE auto reformating that caused it I guess

Reply:
It is this case, the IDE tries to format javadoc param so that each description starts at the same position on its line.
I just let the IDE follow its rules here.